mdata/cache/accnt/flat_accnt.go

```diff
@@ -0,0 +1,130 @@
+package cache
+
+// Flat accounting
+
+type FlatAccnt struct {
```

Comment:
thanks for the improved docs, but to quote the aforementioned url:

> to document a type, variable, constant, function, or even a package, write a regular comment directly preceding its declaration, with no intervening blank line. Godoc will then present that comment as text alongside the item it documents

and:

> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats

so this should be something like : 
(note i removed the bit about the evict loop because that's the concern of CCache)

```
// FlatAccnt implements Flat accounting.
// It keeps track of the chunk cache size and in which order the contained
// chunks have been used the last time. If it detects that the total cache
// size is above the given limit, it feeds the least recently used
// cache chunks into the evict queue.
type FlatAccnt struct {
```